src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Comparator<Employee> employeeComparator = new Comparator<Employee>() {
+            @Override
+            public int compare(Employee employee1, Employee employee2) {
+                int compareAge = Integer.compare(employee1.getAge(), employee2.getAge());
+                if (compareAge != 0) {
+                    return compareAge;
+                }
+                return employee1.getName().compareTo(employee2.getName());
+            }
+        };
+        TreeSet<Employee> sortedEmployee = new TreeSet<>(employeeComparator);
```

Comment:
```suggestion
        Set<Employee> sortedEmployee = new TreeSet<>(employeeComparator);
```
let's use abstraction for references